api/src/test/java/org/apache/iceberg/AssertHelpers.java

```diff
@@ -48,6 +48,22 @@ public static void assertThrows(
     }
   }
 
+  /**
+   * A convenience method to avoid a large number of @Test(expected=...) tests
+   *
+   * @param expected An Exception class that the Runnable should throw
+   * @param containedInMessage A String that should be contained by the thrown exception's message
+   * @param callable A Callable that is expected to throw the exception
+   */
+  public static void assertThrows(
```

Comment:
I think it would be better to use `Assertions` from AssertJ directly (because it gives you greater flexibility) rather than adding new stuff to `AssertHelpers`. And in order to improve the issue around the fully-qualified import due to a conflict with `TableTestBase.Assertions`: I'll open a PR and will rename `TableTestBase.Assertions` to something that doesn't conflict with `Assertions` from AssertJ

Reply:
@hililiwei, what is the purpose of this new method? Just not adding context? Context is a best practice, which is why we require it by not adding helpers that drop it.